src/queue/queue-data-structure.js

```diff
@@ -5,25 +5,48 @@ class Queue {
   }
 
   canEnqueue() {
```

Comment:
Esta función se puede refactorizar a: `return this.stackControl.length < this.MAX_SIZE` ya que esta condición en caso de no estar lleno, va retornar `true`, y si no,  siempre va a retornar `false`